lib/cn/csched_sp3.c

```diff
@@ -1931,6 +1930,22 @@ sp3_tree_shape_check(struct sp3 *sp)
                     lsiz = pcap;
                 }
             }
+
+            readers = atomic_read(&tn->tn_readers);
+            if (readers > 0) {
+                if (!tn->tn_ss_splitting) {
+                    const size_t split_min = (size_t)tree->rp->cn_split_size << 30;
```

Comment:
We should really use size_t for some params imo.

Reply:
I can make these size_t.  Although the bloat the structure they're only used at open and node-split, so they can live at the "cold" end of the rparams struct.  This then requires caller specify the full size...

Reply:
No need to change the parameter types right now. Really just a generic thought I am sending into the void

Reply:
It does not appear that we have a PARAM_TYPE_SIZE...

Reply:
Yes, better than using int64 or long  when you really need size_t.

Reply:
I will get to it this week then.

Reply:
I think a fixed width is better for externally facing params -- it'll be the same width scalar on all builds.